azuredevops/internal/service/serviceendpoint/resource_serviceendpoint_azurerm.go

```diff
@@ -161,6 +170,22 @@ func resourceServiceEndpointAzureRMCreate(d *schema.ResourceData, m interface{})
 		return err
 	}
 
+	if validate {
```

Comment:
can we directly get the `validate` with `d.Get("validate")` instead of `expandServiceEndpointAzureRM`.
`flatten/expand` focuses on the core properties of flattening/expanding resources